CHANGELOG.md

```diff
@@ -17,6 +17,15 @@
   convert that file to plain JS (`//= require active_admin/base` if you
   didn't add any stuff to it).
 
+### Enhancements
+
+* Add your own content to the site `<head>`, like analytics [#5590][] by [@buren][]
+  ```ruby
```

Comment:
Markdownlint is complaining about a missing blank line on top of this line. Can you add it?

Reply:
Will do!